kmip/services/server/engine.py

```diff
@@ -1299,6 +1324,27 @@ def _process_register(self, payload):
 
         return response_payload
 
+    @_kmip_version_supported('1.2')
```

Comment:
No, `1.0` is fine. It's up to the implementation for `Locate` to handle the different possible payload versions.